saffron/src/blob.rs

```diff
@@ -110,11 +114,51 @@ impl<G: KimchiCurve> FieldBlob<G> {
         bytes.truncate(blob.n_bytes);
         bytes
     }
+
+    #[instrument(skip_all, level = "debug")]
+    pub fn update<EFqSponge: Clone + FqSponge<G::BaseField, G, G::ScalarField>>(
+        &mut self,
+        srs: &SRS<G>,
+        domain: &Radix2EvaluationDomain<G::ScalarField>,
+        diff: Diff<G::ScalarField>,
+    ) {
+        let updates: Vec<(usize, PolyComm<G>, DensePolynomial<G::ScalarField>)> = diff
```

Comment:
Let's add also some timers, activated while in debug more. We want to know how fast it is.